Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:buster-slim
```

Comment:
I don't have a specific concerns about this, but I'm curious why this was chosen as the base image.

Reply:
Ease of installing dependencies... Debian stable (stretch) doesn't have node.js 8.x (the LTS).